pages/_app.tsx

```diff
@@ -2,7 +2,7 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { Poppins, Roboto_Mono, Work_Sans } from '@next/font/google';
+import { Poppins, Roboto_Mono, Work_Sans } from 'next/font/google';
```

Comment:
This'll cause build errors please revert to `@next/font/google`

Reply:
yes ill revert it and do the other bug fix as well